crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -0,0 +1,677 @@
+//! Built-in verbs for the Bevy Remote Protocol.
+
+use std::any::TypeId;
+
+use anyhow::{anyhow, Result as AnyhowResult};
+use bevy_ecs::{
+    component::ComponentId,
+    entity::Entity,
+    query::QueryBuilder,
+    reflect::{AppTypeRegistry, ReflectComponent},
+    system::In,
+    world::{EntityRef, EntityWorldMut, FilteredEntityRef, World},
+};
+use bevy_hierarchy::BuildChildren as _;
+use bevy_reflect::{
+    serde::{ReflectSerializer, TypedReflectDeserializer},
+    PartialReflect, TypeRegistration, TypeRegistry,
+};
+use bevy_utils::HashMap;
+use serde::de::DeserializeSeed as _;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+use crate::{error_codes, BrpError, BrpResult};
+
+/// `bevy/get`: Retrieves one or more components from the entity with the given
+/// ID.
+///
+/// The server responds with a [`BrpGetResponse`].
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct BrpGetParams {
+    /// The ID of the entity from which components are to be requested.
+    pub entity: Entity,
+
+    /// The *full paths* of the component types that are to be requested
```

Comment:
By "full paths", are we talking `std::any::type_name` or `TypePath`? I think wherever we mention "type path", "full path", etc. we should have that link to the actual one being used.